libs/game/info.ts

```diff
@@ -358,23 +358,49 @@ namespace info {
         player1.onScore(score, handler);
     }
 
+    /**
+     * Get the value of the current count down
+     */
+    //% block="countdown"
+    //% blockId=gamegetcountdown
+    //% weight=79
+    //% group="Countdown"
+    export function getCountdown() {
+        return (infoState.gameEnd - game.currentScene().millis()) / 1000;
+    }
+
     /**
      * Start a countdown of the given duration in seconds
      * @param duration the duration of the countdown, eg: 10
      */
     //% blockId=gamecountdown block="start countdown %duration (s)"
-    //% help=info/start-countdown weight=79 blockGap=8
+    //% help=info/start-countdown weight=78 blockGap=8
     //% group="Countdown"
     export function startCountdown(duration: number) {
         updateFlag(Visibility.Countdown, true);
         infoState.gameEnd = game.currentScene().millis() + duration * 1000;
         infoState.countdownExpired = false;
     }
 
+    /**
+     * Get the value of the current count down
+     */
+    //% block="change countdown by $seconds (s)"
+    //% blockId=gamechangecountdown
+    //% weight=77
+    //% group="Countdown"
+    export function changeCountdownBy(seconds: number) {
```

Comment:
Infostate should also exist if score or life is used without startcountdown - id just always do `startCountdown((countdown() + seconds) * 1000)` i think